test/foundry/mocks/hts-precompile/HederaNonFungibleToken.sol

```diff
@@ -138,15 +138,6 @@ contract HederaNonFungibleToken is ERC721, Constants {
         return super.transferFrom(from, to, tokenId);
     }
 
-    function safeTransferFrom(address from, address to, uint256 tokenId) public override {
```

Comment:
notice: This is being removed as OZ/contract v5.0.0 only allows override safeTransferFrom(address,address,uint256,bytes), and safeTransaferFrom(address,address,uint256) will cause error `Trying to override non-virtual function.`